recipe/meta.yaml

```diff
@@ -63,7 +64,8 @@ requirements:
     - libpng >=1.6.23,<1.7
     - openblas 0.2.19|0.2.19.*     # [not win]
     - blas 1.1 {{ blas_variant }}  # [not win]
-    - ffmpeg                       # [not win]
+    - ffmpeg >=3.2.3,<3.2.6        # [not win]
```

Comment:
OK - let's get the [pinnings in](https://github.com/conda-forge/conda-forge.github.io/pull/397#issuecomment-309368547) then merge this